examples/elf/Kconfig

```diff
@@ -39,7 +39,7 @@ config EXAMPLES_ELF_EXTERN
 		The file system is assumed to reside on some external media
 		such as an SD card or a USB FLASH drive.  In this case, that
 		external file system must be created manually by copying the
-		files in apps/examples/elf/tests/romfs to the volume.
+		files in apps/examples/elf/tests/extfs to the volume.
```

Comment:
As romfs is used (i.e. `CONFIG_EXAMPLES_ELF_ROMFS`) why change to extfs?

Can you also provide some nuttx/Documentation in this area please? :-)

Reply:
Thanks for the comment.

It is a bit confusing, there are 2 possible uses for romfs: internal (included in nuttx.bin/hex) and external (not in nuttx.bin/hex). In the above option it is using an external romfs (another option is to use external vfat).
The old help message was an error, the files would be found under cromfs.
To correct this error I decided to use romfs and cromfs for internal (in nuttx.bin/hex) fs, and to use extfs for external fs. The external fs can be of type vfat or romfs.

I hope this makes it a little clearer.

Reply:
Thanks @Laczen :-) If we have a full nuttx/doc on that part, which is really interesting, thigs would be clear for everyone I suppose :-)